packages/sync-rules/src/sql_functions.ts

```diff
@@ -251,6 +251,34 @@ const ifnull: DocumentedSqlFunction = {
   detail: 'Returns the first non-null parameter'
 };
 
+// This is the same behavior as the iif function in SQLite
+const iif: DocumentedSqlFunction = {
+  debugName: 'iif',
+  call(x: SqliteValue, y: SqliteValue, z: SqliteValue) {
+    let isTrue: boolean;
+    if (Number.isNaN(x)) {
+      isTrue = false;
+    } else if (typeof x == 'number') {
+      isTrue = x !== 0;
+    } else if (typeof x == 'bigint') {
+      isTrue = x !== 0n;
+    } else {
+      isTrue = false;
+    }
```

Comment:
Could you use the `sqliteBool` function here? That would make sure it's consistent with the boolean handling we have elsewhere.

Reply:
Just did that, didn't see that when implementing the function.